tests/_utils/check_metric_gt_zero

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# parameter 1: port
+# parameter 2: metric name
+# parameter 3: retry count, if check failed we will wait 1s before next retry, until retry time exceeds retry count
+
+set -eu
+
+port=$1
+metric_name=$2
+retry_count=$3
+
+shift 3
+
+counter=0
+while [ $counter -lt $retry_count ]; do
+	metric=$(curl -s http://127.0.0.1:$port/metrics | grep $metric_name | wc -l)
```

Comment:
does this script check `metric_name` exists (may with value `0`), or with a value that greater than zero?

Reply:
this require return metric value greater than 0

Reply:
I guess because `wc -l`, the meaning of `metric` is count not metric value

Reply:
you're right, it's been fixed

Reply:
it's ok to use `INF` but need to think about is number still great than `bash INF(2**31  -1)`